PhysicsTools/PatAlgos/plugins/PATLostTracks.cc

```diff
@@ -194,10 +198,28 @@ void pat::PATLostTracks::produce(edm::StreamID, edm::Event& iEvent, const edm::E
     }
   }
   for (const auto& v0 : *lambdas) {
+    double protonCharge = 0;
     for (size_t dIdx = 0; dIdx < v0.numberOfDaughters(); dIdx++) {
       size_t key = (dynamic_cast<const reco::RecoChargedCandidate*>(v0.daughter(dIdx)))->track().key();
       if (trkStatus[key] == TrkStatus::NOTUSED)
         trkStatus[key] = TrkStatus::VTX;
+      if (xiSelection_)
+        protonCharge += v0.daughter(dIdx)->charge() * v0.daughter(dIdx)->momentum().mag2();
```

Comment:
```suggestion
      protonCharge += v0.daughter(dIdx)->charge() * v0.daughter(dIdx)->momentum().mag2();
```
this part is different from the master version.
Please either drop here or make a follow-up PR in the master branch.

Reply:
Preferred to drop it.